test/unit/rewriteUrlsTest.js

```diff
@@ -8,7 +8,7 @@ import * as rewriteUrls from '../../src/middleware/rewriteUrls'
 import * as utils from '../../src/utils'
 
 // TODO: OHM-699 remove the x prepended to describe when url rewriting is back in support
```

Comment:
Lets remove this comment as well. Now that the logic has been added in, this no longer applies